### ISSUE TITLE
I can't make builds even though it runs in the editor

### ISSUE THREAD

**Issue by Seaglass15** [ababilinski/realsense-apriltags-calibration-unity#2]
Any comments?
![Capture](https://user-images.githubusercontent.com/65552877/161984937-b2411735-c935-4182-9d78-54ba06857410.PNG)

**Comment by ababilinski** [comment]
Good catch! I forgot to update the assembly definition for the RealSense Package Downloader
